File: HackerRank/codetest/ascend.io/truthy.java
class Truthy {
  public static void main(String[] args) {
    if (0) {
      System.out.println("0 was truthy");
    } else {
      System.out.println("0 was falsey");
    }
    if (-1) {
      System.out.println("-1 was truthy");
    } else {
      System.out.println("-1 was falsey");
    }
    if ("") {
      System.out.println("the empty string was truthy");
    } else {
      System.out.println("the empty string was falsey");
    }
    String[] var = null;
    if (var) {
      System.out.println("null was truthy");
    } else {
      System.out.println("an empty array was truthy");
    }
  }
}

File: HackerRank/codetest/ascend.io/go/truthy.go
package main

import "fmt"

func main() {
	if "" {
		fmt.Println("the empty string is truthy")
	} else {
		fmt.Println("the empty string is falsey")
	}
	if []int{} {
		fmt.Println("an empty array is truthy")
	} else {
		fmt.Println("an empty array is falsey")
	}
	if 0 {
		fmt.Println("0 was truthy")
	} else {
		fmt.Println("0 was falsey")
	}
	if nil {

	}
}
